build.cake

```diff
@@ -291,6 +291,18 @@ Task("PackVSTemplates")
     .WithCriteria(() => IsRunningOnWindows())
     .Does(() =>
 {
+    var shortVersion = version;
+	if (shortVersion.Contains("-")) shortVersion = shortVersion.Substring(0, shortVersion.IndexOf("-"));
```

Comment:
Random tab instead of spaces 😅 

Also can be written much shorter but less optimised :P
```cs
var shortVersion = version.Split('-')[0];
```